test/integration/noderesourcetopology_cache_test.go

```diff
@@ -485,7 +485,7 @@ func TestTopologyCachePluginWithoutUpdates(t *testing.T) {
 					checkPod = podIsPending
 				}
 
-				err = checkPod(1*time.Second, 20, cs, p.pod.Namespace, p.pod.Name)
```

Comment:
I'd expect to see a new (sub-)integration test to simulate what was described in the original issue.

Otherwise, I'd postpone the impl. to next release.

Reply:
Have added a new integration test for the case

Reply:
Thanks. But even without introducing the backoff, the integration test seems to be also passing.

```
weih@m1max:~/go/src/sigs.k8s.io/scheduler-plugins|master⚡ ⇒  go test ./test/integration/... -run TestPodgroupBackoff -count 1
ok  	sigs.k8s.io/scheduler-plugins/test/integration	27.100s
weih@m1max:~/go/src/sigs.k8s.io/scheduler-plugins|master⚡ ⇒  gst
On branch master
Your branch is ahead of 'origin/master' by 2 commits.
  (use "git push" to publish your local commits)

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   test/integration/coscheduling_test.go
```

Could you update the test to pass and only pass with the backoff changes? in other words, the test is expected to fail on master.

Reply:
No problem

Reply:
Done